evap/evaluation/models.py

```diff
@@ -453,8 +457,10 @@ def is_user_editor(self, user):
 
     def warnings(self):
         result = []
-        if self.state in ['new', 'prepared', 'editor_approved'] and not self.has_enough_questionnaires:
-            result.append(_("Not enough questionnaires assigned"))
+        if self.state in ['new', 'prepared', 'editor_approved'] and not self.general_contribution_has_questionnaires:
+            result.append(_("General contribution has no questionnaires"))
+        if self.state in ['new', 'prepared', 'editor_approved'] and not self.all_contributions_have_questionnaires:
+            result.append(_("Not all contributions have questionnaires"))
```

Comment:
i would suggest this to prevent both warnings to appear at the same time:
```
if self.state in ['new', 'prepared', 'editor_approved'] and not self.general_contribution_has_questionnaires:
    if not self.all_contributions_have_questionnaires:
        result.append(_("Not all contributions have questionnaires"))
    else:
        result.append(_("General contribution has no questionnaires"))
```